public-html/aboutus.php

```diff
@@ -1,34 +1,13 @@
-<?php 
-require_once($_SERVER["DOCUMENT_ROOT"] . "/../vendor/autoload.php");
+<?php
+require $_SERVER['DOCUMENT_ROOT'] . '/../vendor/autoload.php';
 
-use src\SessionCookie;
-use template\Menu;
+use template\Page;
 
-$session = new SessionCookie;
-
-?>
-<html>
-<title>About us</title>
-
-<head>
-    <?php
-    if (isset($_SESSION["logged_in"])) {
-        $menu = new Menu(basename(__FILE__), $_SESSION["admin"], $_SESSION["uname"]);
-    } else {
-        $menu = new Menu(basename(__FILE__));
-    }
-    ?>
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css">
-    <link rel="stylesheet" href='<?php echo $menu->paths['css'] . "/main.css"; ?>'>
-
-</head>
-<body>
-    <?php
-    $menu->render_header();
-    $menu->render_menu();
-    ?>
-    <div class="main_content">
+$new = new Page;
+$var1 = "Hello Alphons";
+$content = <<<EOT
+<h1> Hi ${var1} </h1>
```

Comment:
Why is this outside `main_content` div?

Reply:
mmm sorry I didnt get you

Reply:
ooo you mean the <h1> tag?
it was for testing, I just removed it!